lib/widgets-page.php

```diff
@@ -29,6 +29,10 @@ function gutenberg_widgets_init( $hook ) {
 			return;
 	}
 
+	wp_add_inline_script(
+		'wp-edit-widgets',
+		'wp.editWidgets.initialize( "widgets-editor" )'
```

Comment:
This should include a terminating semi-colon. Otherwise it may result in other `wp_add_inline_script` for `wp-edit-widgets` being treated as a continuation of this code, e.g. an IIFE would result in "VM662:2 Uncaught TypeError: wp.editWidgets.initialize(...) is not a function".